addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -671,6 +671,63 @@ QUnit.test('chat window: close on ESCAPE', async function (assert) {
     assert.verifySteps(['rpc:channel_fold/closed']);
 });
 
+QUnit.test('focus next visible chat window when closing current chat window with ESCAPE', async function (assert) {
+    assert.expect(4);
+
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [
+                {
+                    channel_type: "channel",
+                    id: 20,
+                    is_minimized: true,
+                    is_pinned: true,
+                    name: "General",
+                    state: "open",
+                    uuid: "channel-20-uuid",
+                },
+                {
+                    channel_type: "channel",
+                    id: 21,
+                    is_minimized: true,
+                    is_pinned: true,
+                    name: "other chat",
+                    state: "open",
+                    uuid: "channel-21-uuid",
+                },
+            ],
+        },
+    });
+    await this.start();
+    assert.containsN(
+        document.body,
+        ".o_ChatWindow .o_ComposerTextInput_textarea",
+        2,
+        "2 chat windows should be present"
```

Comment:
Same remark concerning single & double quotes. Selector is technical string, so `'` (exception when needing string interpolation, could use ES6 string template (`)). Assertion comment is UI string so `"`.